ete3/tools/ete_evol.py

```diff
@@ -439,6 +447,16 @@ def remove_duplicated_marks(nodes, marks, tree):
         del(marks[bad])
         del(nodes[bad])
 
+
+def name_model(tree, base_name):
+    """
+    transform the name string into summary of its name and a digestion of the
+    full name
+    """
+    return base_name[:12] + '~' + md5(tree.get_topology_id(attr="name") +
```

Comment:
ok, sorry. I do a md5 of an md5 to include the model name into the hash, not only the tree_topology.